packages/core/content-manager/admin/src/pages/ListView/components/BulkActions/PublishAction.tsx

```diff
@@ -248,18 +248,10 @@ const SelectedEntriesTableContent = ({
                     search: row.locale && `?plugins[i18n][locale]=${row.locale}`,
                   }}
                   state={{ from: pathname }}
-                  label={formatMessage(
-                    { id: 'app.component.HelperPluginTable.edit', defaultMessage: 'Edit {target}' },
-                    {
-                      target: formatMessage(
-                        {
-                          id: 'content-manager.components.ListViewHelperPluginTable.row-line',
-                          defaultMessage: 'item line {number}',
-                        },
-                        { number: index + 1 }
-                      ),
-                    }
-                  )}
+                  label={formatMessage({
+                    id: 'app.component.HelperPluginTable.edit',
```

Comment:
Can I ask you to add the id also in the translation file packages/core/content-manager/admin/src/translations/en.json ? 🙏🏽

Reply:
@simotae14 Rather adding new id i used the old one for Edit msg